typings/query-helpers.d.ts

```diff
@@ -4,12 +4,12 @@ import { Matcher, MatcherOptions } from './matches';
 
 type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
-export interface SelectorMatcherOptions extends MatcherOptions {
+export type SelectorMatcherOptions = Omit<MatcherOptions, 'selector'> & {
   selector?: string;
-}
+};
 
 type ReactTestInstance = {
-  getProp: (string) => NativeTestInstance;
+  getProp: (str: string) => NativeTestInstance;
```

Comment:
Can you make this something more descriptive of what this is? Like `name` possibly?

Reply:
Yep, can we make the name of the argument `name` so that it’s clear from the typings what `getProp` is looking for? ☺️